airbyte-cdk/python/airbyte_cdk/utils/__init__.py

```diff
@@ -1,6 +1,7 @@
 #
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
+from .schema_inferrer import SchemaInferrer
```

Comment:
need an extra space between the licence and import or our CI formatting checks will fail